### ISSUE TITLE
Need to call setup with a colon

### ISSUE THREAD

**Issue by yggdr** [mnowotnik/noteflow.nvim#6]
I just had a bit of a headache with setting up noteflow, because it complained about an nil valued `opts` in `config.setup`, despite me giving it a table. It took a while until I realised you defined `M:setup`, not `M.setup`, so you also have to call it as `require'noteflow':setup{vault_dir=…}`, with a colon instead of a dot like you did in the readme.

**Comment by mnowotnik** [comment]
You're right. i forgot to update readme after refactoring. Do not hesitate to create more issues if you encounter other problems.
